Dockerfile

```diff
@@ -62,8 +62,6 @@ WORKDIR /var/www/html
 # COPY docker/*.php /var/www/html/app/config/production/
 COPY docker/docker.env /var/www/html/.env
```

Comment:
This seems great! Would we need to also do it here, though? The later `chown` seems like it would grab this `.env` file as well?

Let me know about that (or add it, if needed) and I'll get it approved for you right away!

Reply:
I'll add a commit to the PR.

Reply:
Hi there! Would you still be able to make this change?